libs/brain/tooltip/src/lib/brn-tooltip-trigger.directive.ts

```diff
@@ -688,9 +623,14 @@ export class BrnTooltipTriggerDirective implements OnDestroy, AfterViewInit {
 	}
 
 	/** Binds the pointer events to the tooltip trigger. */
-	private _setupPointerEnterEventsIfNeeded() {
+	private _setupPointerEnterEventsIfNeeded(): void {
 		// Optimization: Defer hooking up events if there's no content or the tooltip is disabled.
-		if (this._disabled() || !this.content || !this._viewInitialized || this._passiveListeners.length) {
+		if (
+			this.brnTooltipDisabled() ||
+			!this.brnTooltipTriggerState ||
```

Comment:
Are we missing the parantheses () here?
```suggestion
			!this.brnTooltipTriggerState() ||
```

Reply:
yes you are right, thank you. I changed it :)